src/ManifoldsBase.jl

```diff
@@ -356,16 +356,11 @@ Distance ``d`` such that
 is injective for all tangent vectors shorter than ``d`` (i.e. has an inverse) for point `p`
 if provided or all manifold points otherwise.
 """
-function injectivity_radius(M::AbstractManifold)
-    return error(manifold_function_not_implemented_message(M, injectivity_radius))
-end
+injectivity_radius(M::AbstractManifold)
```

Comment:
Shouldn't we split `injectivity_radius` on the second level similartly to `retract`? That is have `injectivity_radius_exponential`, `injectivity_radius_qr` etc.? This would save us some ambiguities.

Reply:
Yes, I thought that as well when I saw that - it would fit our deign better.